picturae_csv_create.py

```diff
@@ -287,6 +287,18 @@ def col_clean(self):
         for col_string in date_col_list:
             self.record_full[col_string] = pd.to_datetime(self.record_full[col_string],
                                                           format='%m/%d/%Y').dt.strftime('%Y-%m-%d')
+
```

Comment:
adds placeholder rank of "subsp."  to first Rank 1 if missing before running through TNRS, for relatively uncommon but convoluted taxonomic names with multiple missing infraspecific ranks that are missing rank,  the plan is to flag those ahead of time with a simple script, and correct manually.